hollow/src/main/java/com/netflix/hollow/api/producer/WriteStateImpl.java

```diff
@@ -26,9 +28,12 @@
  * @author Tim Taylor {@literal<tim@toolbear.io>}
  */
 final class WriteStateImpl implements HollowProducer.WriteState {
+    private static final String LATE_MODIFICATION_MESSAGE =
+            "attempt to modify state after populate stage complete; version=%d";
     private final long version;
-    private final HollowObjectMapper objectMapper;
-    private final HollowProducer.ReadState priorReadState;
```

Comment:
Why is final removed?

Reply:
Oh, that's on accident. At one point I had it implementing `AutoCloseable` and I was nulling those out. I reverted that bit, but forgot to bring back `final`.